splunk-apm/README.md

```diff
@@ -5,70 +5,95 @@ Splunk OpenTelemetry Lambda Layer for AWS lambda enables seamless instrumentatio
 ## Get started 
 
 Currently, following components are supported:
-- [Java wrapper](#java-wrapper) 
+- [Java wrapper](#java-wrapper)
+- [Python wrapper](#python-wrapper)  
 - [Metrics extension](#metrics-extension)
 
 ### Configuration
 
-All wrapper layers are preconfigured to use Splunk as the tracing backend.
+All components are preconfigured to use Splunk as the tracing backend.
```

Comment:
```suggestion
All components are preconfigured to use Splunk as the tracing back end.
```

Reply:
:O It has always been "frontend" and "backend" (without spaces).

Reply:
Non in the splunk docs guidelines... https://docs.splunk.com/Documentation/StyleGuide/current/StyleGuide/Usagedictionary#B